### PR TITLE
add jquery-ajaxQueue v0.1.1 with autoupdate config

### LINKED ISSUE #7601: [Request] Add ajaxQueue
**Library name:** ajaxQueue
**Git repository url:** https://github.com/gnarf/jquery-ajaxQueue

### PR DESCRIPTION
**Pull request by dannyxx001**
PR for #7601 
@kennynaoh please help me review this PR, thanks.
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/gnarf/jquery-ajaxQueue
 * Official website (optional, not the repository):
 * NPM package url (optional):
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 124
   - Count of watchers: 12
   - Count of forks: 33
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #7601, cc @gnarf

### CODE REVIEW

File: ajax/libs/jquery-ajaxQueue/package.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": "jquery-ajaxQueue",
+  "filename": "jquery.ajaxQueue.min.js",
+  "title": "jQuery Ajax Queue",
+  "description": "A simple queue for ajax requests",
+  "version": "0.1.1",
+  "author": {
+    "name": "Corey Frang",
+    "email": "gnarf37@gmail.com",
+    "url": "https://github.com/gnarf37"
```
Comment (kennynaoh) [inline ajax/libs/jquery-ajaxQueue/package.json]: url should be updated

Comment (dannyxx001) [inline ajax/libs/jquery-ajaxQueue/package.json]: OK, I had updated it, @kennynaoh please help me review again, thank you.

**Comment by kennynaoh** [comment]
@PeterDaveHello 
I think this PR is correct
Please help me review it again
Thanks

File: ajax/libs/jquery-ajaxQueue/package.json (hunk 1 above)
Comment (PeterDaveHello) [inline ajax/libs/jquery-ajaxQueue/package.json]: :+1:

**Pull request merged**